verify.py

```diff
@@ -96,7 +96,7 @@ def __init__(self, configfile, auth):
             elif opts[line] == '-d':
                 self.desc = opts[line + 1]
             elif opts[line] == '-b':
-                self.bin = opts[line + 1]
+                self.bin = self.desc
```

Comment:
Thanks, @dbernstein! So the problem for us is that if the -b comes before the -d in the config file, the verification tool won't be able to correctly set the location of the binaries because the location of the desc directory won't yet have been set.